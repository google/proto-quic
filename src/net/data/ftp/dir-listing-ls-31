drwxrwxrwx    6 1000     1000         4096 Jul 23  2011 Alfresco
drwxrwxrwx    2 1000     1000         4096 Mar 18  2010 DIRECTUM
-rwxrwxrwx    1 1000     1000       222510 Mar 29  2010 Featurelist 6.30.pdf
drwxrwxrwx    2 1000     1000         4096 Jul 23  2011 NauDoc_v4.4
-rwxrwxrwx    1 1000     1000      1564767 Apr 06  2010 RUS_v_01_00_МЕТОД СБОРА И ДОКУМЕНТИРОВАНИЯ
     ТРЕБОВАНИЙ К ПОРТАЛУ
.pdf
drwxrwxrwx    4 1000     1000         4096 Jul 22  2011 Videoconferencing
drwxrwxrwx    3 1000     1000         4096 Apr 15  2010 Virtualization
-rwxrwxrwx    1 1000     1000     111726333 Jan 10  2010 electr_docoborot_2010.flv
-rwxrwxrwx    1 1000     1000      4224387 Mar 31  2010 millenniumbsa.pdf
drwxrwxrwx    5 1000     1000         4096 Apr 16  2010 Бизнес План
-rwxrwxrwx    1 1000     1000       138217 Apr 16  2010 Мониторинг в инфраструктуре распределенных приложений .NET.rar
-rwxrwxrwx    1 1000     1000         4131 Feb 25  2010 О законе О персональных данных.txt
-rwxrwxrwx    1 1000     1000      3627173 Feb 21  2010 Шеер А.В. -- Бизнес-процессы. Основные понятия..djvu
