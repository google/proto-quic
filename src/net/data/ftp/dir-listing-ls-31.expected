d
Alfresco
-1
2011
7
23
0
0

d
DIRECTUM
-1
2010
3
18
0
0

-
Featurelist 6.30.pdf
222510
2010
3
29
0
0

d
NauDoc_v4.4
-1
2011
7
23
0
0

-
RUS_v_01_00_МЕТОД СБОРА И ДОКУМЕНТИРОВАНИЯ
     ТРЕБОВАНИЙ К ПОРТАЛУ
.pdf
1564767
2010
4
6
0
0

d
Videoconferencing
-1
2011
7
22
0
0

d
Virtualization
-1
2010
4
15
0
0

-
electr_docoborot_2010.flv
111726333
2010
1
10
0
0

-
millenniumbsa.pdf
4224387
2010
3
31
0
0

d
Бизнес План
-1
2010
4
16
0
0

-
Мониторинг в инфраструктуре распределенных приложений .NET.rar
138217
2010
4
16
0
0

-
О законе О персональных данных.txt
4131
2010
2
25
0
0

-
Шеер А.В. -- Бизнес-процессы. Основные понятия..djvu
3627173
2010
2
21
0
0
